CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG for Sulu
 ==================
 
+* dev-master
+    * HOTFIX     #3573 [RouteBundle]             Added rawurldecode to decode path-info before searching for route
+    * HOTFIX     #3573 [ContentBundle]           Added rawurldecode to decode path-info before searching for route
+    * HOTFIX     #3573 [CustomUrlBundle]         Added rawurldecode to decode path-info before searching for route
```

Comment:
k?

Reply:
I think that's how we've done it for already several times now, if the PR touches multiple bundles.

Reply:
didn't see that it were different bundles 🙈